users/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=8, required=True)
+
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'password', 'first_name', 'last_name', 'phone']
+
+    def create(self, validated_data):
+        user = User(
+            email=validated_data["email"],
+            first_name=validated_data.get("first_name", ""),
+            last_name=validated_data.get("last_name", ""),
+            phone=validated_data.get("phone", "")
```

Comment:
Use `phonenumber_field.serializer.PhoneNumberField` to ensure phone number validity.